Sazabi.rc

```diff
@@ -2322,7 +2322,7 @@ BEGIN
     IDS_STRING_CEF_LOCALE   "en"
     IDS_STRING_CEF_ACCEPT_LANGUAGE_LIST "en-US"
     IDS_STRING_PROPRIETARY_CODEC_LICENSES 
-                            "\n\nAVC\nTHIS PRODUCT IS LICENSED UNDER THE\nAVC PATENT PORTFOLIO LICENSE FOR THE PERSONAL USE OF A\nCONSUMER OR OTHER USES IN WHICH IT DOES NOT RECEIVE\nREMUNERATION TO (i) ENCODE VIDEO IN COMPLIANCE WITH THE\nAVC STANDARD (�AVC VIDEO�) AND/OR (ii) DECODE AVC VIDEO\nTHAT WAS ENCODED BY A CONSUMER ENGAGED IN A PERSONAL\nACTIVITY AND/OR WAS OBTAINED FROM A VIDEO PROVIDER\nLICENSED TO PROVIDE AVC VIDEO. NO LICENSE IS GRANTED OR\nSHALL BE IMPLIED FOR ANY OTHER USE. ADDITIONAL\nINFORMATION MAY BE OBTAINED FROM MPEG LA, L.L.C. SEE\nHTTP://WWW.MPEGLA.COM"
+                            "\n\nAVC\nTHIS PRODUCT IS LICENSED UNDER THE\nAVC PATENT PORTFOLIO LICENSE FOR THE PERSONAL USE OF A\nCONSUMER OR OTHER USES IN WHICH IT DOES NOT RECEIVE\nREMUNERATION TO (i) ENCODE VIDEO IN COMPLIANCE WITH THE\nAVC STANDARD (�AVC VIDEO�E AND/OR (ii) DECODE AVC VIDEO\nTHAT WAS ENCODED BY A CONSUMER ENGAGED IN A PERSONAL\nACTIVITY AND/OR WAS OBTAINED FROM A VIDEO PROVIDER\nLICENSED TO PROVIDE AVC VIDEO. NO LICENSE IS GRANTED OR\nSHALL BE IMPLIED FOR ANY OTHER USE. ADDITIONAL\nINFORMATION MAY BE OBTAINED FROM MPEG LA, L.L.C. SEE\nHTTP://WWW.MPEGLA.COM"

```

Comment:
This is an unintended change.
I will revert this.

Reply:
I found that this is a bug.
Fixing in another PR.

https://github.com/ThinBridge/Chronos/pull/103